torch_geometric/utils/to_dgnn.py

```diff
@@ -0,0 +1,31 @@
+import scipy.sparse as sp
+import torch
+
+
+def to_dgnn(edge_index):
```

Comment:
Would be also good to name it more generally, e.g., `compute_all_graph_representations`.